pandas/tests/frame/test_replace.py

```diff
@@ -1070,17 +1070,27 @@ def test_replace_truthy(self):
         assert_frame_equal(r, e)
 
     def test_replace_int_to_int_chain(self):
+        # GH 27660 keep behaviour consistent for simple dictionary and
+        # nested dictionary replacement
         df = DataFrame({"a": list(range(1, 5))})
-        with pytest.raises(ValueError, match="Replacement not allowed .+"):
-            df.replace({"a": dict(zip(range(1, 5), range(2, 6)))})
+
+        # nested dictionary replace
```

Comment:
These comments don't add much so I think OK to remove (here and below)

Reply:
removed!